manifests/backend.pp

```diff
@@ -77,6 +82,8 @@
     $instance_name = "haproxy-${instance}"
     $config_file = inline_template($haproxy::params::config_file_tmpl)
   }
+  include haproxy::globals
+  $_sort_options_alphabetic = pick($sort_options_alphabetic, $haproxy::globals::sort_options_alphabetic)
```

Comment:
Is it impossible to put the `sort_options_alphabetic` parameter in the base `haproxy` class and follow this pattern as usual, instead of having a `haproxy::globals` class? We only use globals when this is impossible.